server/1.2/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM fabric8/java-centos-openjdk11-jdk
+
+LABEL maintainer="Debezium Community"
+
+#
+# Set the version, home directory, and MD5 hash.
+#
+ENV DEBEZIUM_VERSION="1.2.1.Final" \
+    SERVER_HOME=/debezium \
+    SERVER_MD5=3f39de479ba7fc90316d127aa9b5f7d5
+ENV SERVER_URL_PATH=io/debezium/debezium-server-dist/$DEBEZIUM_VERSION/debezium-server-dist-$DEBEZIUM_VERSION.tar.gz
+
+#
+# Create a user and home directory for Debezium Server
+#
+USER root
+RUN groupadd -r debezium -g 1001 && \
+    useradd -u 1001 -r -g debezium -m -d $SERVER_HOME -s /sbin/nologin -c "Debezium Server user" debezium && \
```

Comment:
is a static group id and user id mandatory?

Reply:
Yes, there is a (small) risk that with every image the user will be updated and then data created in columes will use different users.